### PR TITLE
Minor Change to Statement on Quay.io Vulnerabilities

### PR DESCRIPTION
**Pull request by paolaperaza**
Hey @kwcanuck - since our statements on Quay.io CVEs are decently sensitive, I went back and re-read them this morning and had some additional suggestions. Notably:

- The report on Quay.io isn't properly called `Common Vulnerabilities and Exposures` -- technically on Quay, it's just called **Security Scan**, so I proposed to change the title of this section to match what a user would see
- I split up the paragraphs in this section for readability and left the user CTA (contact support) at the end
- I also linked to Project Quay (which provides these results) as well as our Astro Runtime repo, which is where people would see these results
- "High-level vulnerabilities" I don't think is an industry standard term -- replaced with `critical` 

image

I know we'll be modifying this doc soon so I didn't do much else other than minor copy suggestions. Let me know what you think.

### CODE REVIEW

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -72,10 +72,13 @@ When Astronomer identifies a significant security vulnerability in Astro Runtime
 
 Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
 
-### The Quay Common Vulnerabilities and Exposures Report
+### Security Scan Results on Quay
 
-Astronomer is aware of the Quay Common Vulnerabilities and Exposures (CVE) report and monitors it frequently to determine if the vulnerabilities identified in the report pose a risk to organizations using Astro Runtime images. Astronomer works with vendors to correct vulnerabilities and regularly adds fixes to stable and LTS releases. If there is a high-level vulnerability in the CVE report that is causing concern for your organization, contact [Astronomer Support](https://support.astronomer.io/).
+Astronomer is aware of the **Security Scan** results that are provided by [Project Quay](https://www.projectquay.io/) for each Astro Runtime image and are publicly available on [Quay.io](https://quay.io/repository/astronomer/astro-runtime?tab=tags).
```
Comment (kwcanuck) [inline astro/runtime-version-lifecycle-policy.md:77]: Hi, @paolaperaza. 

You indicated _The report on Quay.io isn't properly called `Common Vulnerabilities and Exposures` -- technically on Quay, it's just called **Security Scan**, so I proposed to change the title of this section to match what a user would see. _  Are we referencing the same report? The report I was referencing contained a pie chart and a list of vulnerabilities (if I recall). CVE, is short for Common Vulnerabilities and Exposures, and it's is a list of publicly disclosed computer security flaws. I can't seem to locate the conversation where the image of the report was provided...

File: astro/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -90,19 +93,19 @@ For each `major.minor` pair, only the latest patch is supported at any given tim
...
+- Astronomer is not obligated to answer questions regarding a Deployment that is running that version.
 - New Deployments cannot be created on Astro with that version of Runtime. Versions that are no longer maintained will not render as an option in the Deployment creation process from the Cloud UI, API, or Astro CLI.
 - The Deployment view of the Cloud UI will show a warning that encourages the user to upgrade if the Deployment is running that version.
 - The latest version of the Astro CLI will show a warning if a user pushes a Docker image to Astronomer that corresponds to that version.
 
-To ensure reliability, service will not be interrupted for Deployments running a version of Runtime that is no longer supported. Unsupported versions will also continue to be available for local development and testing via the Astro CLI.
+To ensure reliability, service will not be interrupted for Deployments running a version of Astro Runtime that is no longer in maintenance. Unsupported versions will also continue to be available for local development and testing via the Astro CLI.
```
Comment (kwcanuck) [inline astro/runtime-version-lifecycle-policy.md:108]:
```suggestion
Astronomer will not interrupt service for Deployments running Astro Runtime versions that are no longer in maintenance. Unsupported Astro Runtime versions are available for local development and testing with the Astro CLI.
```

File: astro/runtime-version-lifecycle-policy.md [hunk 3]
```diff
@@ -90,19 +93,19 @@ For each `major.minor` pair, only the latest patch is supported at any given tim
...
 - Astronomer will regularly publish bug or security fixes identified as high priority.
-- The Astronomer Support team will offer support for paying customers running a supported version of Runtime via the [Astronomer Support Portal](https://support.astronomer.io).
-- A user can create a new Deployment via the Cloud UI, API, or Astro CLI with any supported `major.minor` version pair of Runtime. For new Deployments, the Cloud UI assumes the latest patch.
+- Astronomer will offer support for paying customers running a supported version of Runtime via [Astronomer Support](https://support.astronomer.io).
+- A user can create a new Deployment with the Cloud UI, API, or Astro CLI with any supported `major.minor` version pair of Runtime. For new Deployments, the Cloud UI assumes the latest patch.
 
 When the maintenance window for a given version of Runtime ends, the following is true:
 
-- The Astronomer Support team is not obligated to answer questions regarding a Deployment that is running that version.
+- Astronomer is not obligated to answer questions regarding a Deployment that is running that version.
```
Comment (kwcanuck) [inline astro/runtime-version-lifecycle-policy.md:103]:
```suggestion
- Astronomer is not obligated to answer questions regarding a Deployment that is running an unsupported version.
```

File: astro/runtime-version-lifecycle-policy.md (hunk 1 above)
Comment (paolaperaza) [reply astro/runtime-version-lifecycle-policy.md:77]: @kwcanuck Per our live conversation after this, we agreed to call this "section" in Quay the `Security Scan Report` given that's what it's called in their UI:

image

We can refer to individual vulnerabilities as CVEs -- as you point out, that is the technically accurate language

**Review by paolaperaza — COMMENTED** [review]
@kwcanuck I think this is ready for you! Want to take a final pass?

Left one comment re: the "CVE" vs. "Security Scan" terminology. Applied your suggestions with a minor tweak.

**Comment by kwcanuck** [comment]
@paolaperaza and @jwitz all edits requested to date have been added to the doc. It's ready for final review.

**Comment by kwcanuck** [comment]
@paolaperaza and @jwitz all edits requested to date have been added to the doc. It's ready for final review.

**Review by kwcanuck — APPROVED** [review]

File: astro/runtime-version-lifecycle-policy.md [hunk 4]
```diff
@@ -89,19 +93,19 @@ For each `major.minor` pair, only the latest patch is supported at any given tim
 
 Within the maintenance window of each Astro Runtime version, the following is true:
 
-- A set of Docker images corresponding to that version are available for download via [Quay.io](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
+- A set of Docker images corresponding to that version are available for download on [Quay.io](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
```
Comment (paolaperaza) [inline astro/runtime-version-lifecycle-policy.md:96]:
```suggestion
- A set of Docker images corresponding to that version are available for download on [Quay](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
```

File: astro/runtime-version-lifecycle-policy.md [hunk 5]
```diff
@@ -72,9 +72,13 @@ When Astronomer identifies a significant security vulnerability in Astro Runtime
 
 Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
 
-### The Quay Common Vulnerabilities and Exposures Report
+### Security scan results on Quay
```
Comment (paolaperaza) [inline astro/runtime-version-lifecycle-policy.md:75]:
```suggestion
### Security scan results on Quay.io
```

File: astro/runtime-version-lifecycle-policy.md [hunk 6]
```diff
@@ -89,19 +93,19 @@ For each `major.minor` pair, only the latest patch is supported at any given tim
 
 Within the maintenance window of each Astro Runtime version, the following is true:
 
-- A set of Docker images corresponding to that version are available for download via [Quay.io](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
+- A set of Docker images corresponding to that version are available for download on [Quay](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
```
Comment (paolaperaza) [inline astro/runtime-version-lifecycle-policy.md:96]:
```suggestion
- A set of Docker images corresponding to that version are available for download on [Quay.io](https://quay.io/repository/astronomer/astro-runtime?tab=tags) and PyPi.
```

**Comment by paolaperaza** [comment]
Thank you @kwcanuck !

**Pull request merged**